src/CellData/CellData.jl

```diff
@@ -57,6 +61,10 @@ export CellDof
 export get_normal_vector
 export get_cell_measure
 
+export make_inverse_table
+export point_to_cell!
```

Comment:
You are exporting `point_to_cell!` but the function to generate its cache object seems to be private.

Here, there are 2 options:

1. Do not export `point_to_cell!` and rename it to  `_point_to_cell!` so that it is clear that it's private.
2.  Make public the also the mechanism to generate the cache. In this case, follow the API of the abstract type `Map` just to be consistent with other parts of the code. 
```julia
struct PointToCellMap{T<:Triangulation} <: Map
  trian::T
end

return_cache(k::PointToCellMap,x::Point) = ...
evaluate!(cache,k::PointToCellMap,x::Point) = ...
```